File: public/phase-2-20160620022203/JAVASCRIPT/junior-broker.js
// Name : Junior Broker
// input:
// n :  number of days
// e :  amount of Euro Finn has on the first day
// a :  the currency rate of Dollar to Euro on day i
var index,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var e = parseInt(elements[1], 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));


// write your code here
// store your results in `result`

// Dummy Data
result = 8;


// output
print(result);

File: public/phase-2-20160620022203/JAVA/junior-broker.java
// Name : Junior Broker
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  number of days
		// e :  amount of Euro Finn has on the first day
		// a :  the currency rate of Dollar to Euro on day i
		int index;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int e = Integer.parseInt(elements[1]);
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=8;

		System.out.println(result);

	}
}